tastypie/bundle.py

```diff
@@ -32,4 +33,7 @@ def __init__(self,
         self.via_uri = via_uri
 
     def __repr__(self):
-        return "<Bundle for obj: '%s' and with data: '%s'>" % (self.obj, self.data)
+        repr_string = "<Bundle for obj: '%s' and with data: '%s'>"
```

Comment:
Could you make this a unicode string (u""), perform the substitution, and then perform encode('utf-8') if six.PY2? Otherwise, if self.obj or self.data are unicode, the returned string could be unicode as well. It's unlikely, but better to be safe.